src/test/java/org/fundacionjala/coding/wilson/PlanetTwisted37Test.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.wilson;
+import java.util.Arrays;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author wiltrouble
+ */
+
+public final class PlanetTwisted37Test {
+    private static final Integer[] N1 = {1, 2, 3, 4, 5, 6, 7, 8, 9};
+    private static final Integer[] EXPECT1 = {1, 2, 7, 4, 5, 6, 3, 8, 9};
+    private static final Integer[] N2 = {9, 2, 4, 7, 3};
+    private static final Integer[] EXPECT2 = {2, 7, 4, 3, 9};
+    private static final Integer[] N3 = {12, 13, 14};
+    private static final Integer[] EXPECT3 = {12, 14, 13};
```

Comment:
Don't create constants 
You should create final local variables inside test method.